### DISCUSSION TITLE
Risk Adjusted Return Aggregated Indicator

### DISCUSSION THREAD

**Discussion by mikey-joyce** [kernc/backtesting.py#905] [Ideas]
I am creating a Mamdani system that aggregates Sharpe ratio, Sortino ratio, and Calmar ratio for my project and gives an analysis. I was curious if I submit a pull request if there is any interest in this being merged to the project?

**Comment by kernc** [comment]
I think there is sufficient interest at least to see and understand what you're talking about. I, for one, have no idea what a Mamdani system is. 😅

**Comment by mikey-joyce** [comment]
The following link provides a short explanation over Mamdani systems https://www.mathworks.com/help/fuzzy/types-of-fuzzy-inference-systems.html.

Essentially it would be able to give a general analysis over risk adjusted returns. I would probably interpret this as a classification problem so the output would be something like: Very Risky, Risky, Neutral, etc... It could give a brief indication on whether or not what you are doing is working (to some extent of course) without having to fully understand the nuances between each ratio. Could be good for beginners?
